pkg/api/v1beta1/types.go

```diff
@@ -1067,9 +1071,9 @@ type ResourceQuotaSpec struct {
 // ResourceQuotaStatus defines the enforced hard limits and observed use
 type ResourceQuotaStatus struct {
 	// Hard is the set of enforced hard limits for each named resource
-	Hard ResourceList `json:"hard,omitempty"`
+	Hard ResourceList `json:"hard,omitempty" description:"hard is the set of enforced hard limits for each named resource"`
 	// Used is the current observed total usage of the resource in the namespace
-	Used ResourceList `json:"used,omitempty"`
+	Used ResourceList `json:"used,omitempty" description:"used is the current observed total usage of the resource in the namespace"`
```

Comment:
Anyway, you get the idea. Omit the field name, "is", and "the". 

"Defines"/"specifies" and "describes" might sometimes be appropriate to distinguish spec and status, but might not always be necessary.